composer.json

```diff
@@ -28,7 +28,6 @@
         "ext-xsl": "*",
         "ext-zip": "*",
         "lib-libxml": "*",
-        "braintree/braintree_php": "3.35.0",
```

Comment:
It breaks Git Magento installation as Composer installs everything from the root composer.json.